diff --git a/node_modules/desktop-trampoline/index.js b/node_modules/desktop-trampoline/index.js
index 0891bae..541c666 100644
--- a/node_modules/desktop-trampoline/index.js
+++ b/node_modules/desktop-trampoline/index.js
@@ -1,12 +1,10 @@
+const os = require('os');
 const Path = require('path')
 
+const  prebuildTargetPath =  () => Path.dirname(require.resolve(`@hackolade/desktop-trampoline-${os.platform()}-${os.arch()}`));
+
 function getDesktopTrampolinePath() {
-  return Path.join(
-    __dirname,
-    'build',
-    'Release',
-    getDesktopTrampolineFilename()
-  )
+  return Path.join( prebuildTargetPath(),getDesktopTrampolineFilename());
 }
 
 function getDesktopTrampolineFilename() {
@@ -16,7 +14,7 @@ function getDesktopTrampolineFilename() {
 }
 
 function getSSHWrapperPath() {
-  return Path.join(__dirname, 'build', 'Release', getSSHWrapperFilename())
+  return Path.join(prebuildTargetPath(), getSSHWrapperFilename());
 }
 
 function getSSHWrapperFilename() {
diff --git a/node_modules/desktop-trampoline/package.json b/node_modules/desktop-trampoline/package.json
index 0cc0262..82eab84 100644
--- a/node_modules/desktop-trampoline/package.json
+++ b/node_modules/desktop-trampoline/package.json
@@ -10,8 +10,7 @@
   },
   "types": "./index.d.ts",
   "scripts": {
-    "build": "node-gyp rebuild",
-    "install": "prebuild-install || node-gyp rebuild",
+    "install": "prebuild-install",
     "test": "jest",
     "lint": "prettier -c **/*.js **/*.md",
     "lint:fix": "prettier --write **/*.js **/*.md",
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-arm64/desktop-trampoline b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/desktop-trampoline
new file mode 100755
index 0000000..c7e3356
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/desktop-trampoline differ
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-arm64/ssh-wrapper b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/ssh-wrapper
new file mode 100755
index 0000000..ac3dd7d
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/ssh-wrapper differ
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-x64/desktop-trampoline b/node_modules/desktop-trampoline/prebuilds/darwin-x64/desktop-trampoline
new file mode 100755
index 0000000..5d20e45
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-x64/desktop-trampoline differ
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-x64/ssh-wrapper b/node_modules/desktop-trampoline/prebuilds/darwin-x64/ssh-wrapper
new file mode 100755
index 0000000..b174748
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-x64/ssh-wrapper differ
diff --git a/node_modules/desktop-trampoline/prebuilds/linux-x64/desktop-trampoline b/node_modules/desktop-trampoline/prebuilds/linux-x64/desktop-trampoline
new file mode 100755
index 0000000..a349686
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/linux-x64/desktop-trampoline differ
diff --git a/node_modules/desktop-trampoline/prebuilds/linux-x64/ssh-wrapper b/node_modules/desktop-trampoline/prebuilds/linux-x64/ssh-wrapper
new file mode 100755
index 0000000..3fd2c72
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/linux-x64/ssh-wrapper differ
diff --git a/node_modules/desktop-trampoline/prebuilds/win32-x64/desktop-trampoline.exe b/node_modules/desktop-trampoline/prebuilds/win32-x64/desktop-trampoline.exe
new file mode 100644
index 0000000..6dc249f
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/win32-x64/desktop-trampoline.exe differ
diff --git a/node_modules/desktop-trampoline/prebuilds/win32-x64/ssh-wrapper.exe b/node_modules/desktop-trampoline/prebuilds/win32-x64/ssh-wrapper.exe
new file mode 100644
index 0000000..3d89728
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/win32-x64/ssh-wrapper.exe differ
