diff --git a/node_modules/desktop-trampoline/README.md b/node_modules/desktop-trampoline/README.md
index f01f067..146f825 100644
--- a/node_modules/desktop-trampoline/README.md
+++ b/node_modules/desktop-trampoline/README.md
@@ -1,3 +1,10 @@
+# Repackaged version of Desktop Trampoline
+
+This is a convenience package that will install all prebuilds for all supported platforms, to the contrary to the upstream package that constraint the installation only to the runtime platform (os/cpu).
+This typically enable cross building an Electron application.
+
+The package is cleaned up to only embbed the necessary.
+
 # Desktop Trampoline
 
 A cross-platform no-dependency C executable trampoline which lets GitHub Desktop
diff --git a/node_modules/desktop-trampoline/index.js b/node_modules/desktop-trampoline/index.js
index 0891bae..541c666 100644
--- a/node_modules/desktop-trampoline/index.js
+++ b/node_modules/desktop-trampoline/index.js
@@ -1,12 +1,10 @@
+const os = require('os');
 const Path = require('path')
 
+const  prebuildTargetPath =  () => Path.dirname(require.resolve(`@hackolade/desktop-trampoline-${os.platform()}-${os.arch()}`));
+
 function getDesktopTrampolinePath() {
-  return Path.join(
-    __dirname,
-    'build',
-    'Release',
-    getDesktopTrampolineFilename()
-  )
+  return Path.join( prebuildTargetPath(),getDesktopTrampolineFilename());
 }
 
 function getDesktopTrampolineFilename() {
@@ -16,7 +14,7 @@ function getDesktopTrampolineFilename() {
 }
 
 function getSSHWrapperPath() {
-  return Path.join(__dirname, 'build', 'Release', getSSHWrapperFilename())
+  return Path.join(prebuildTargetPath(), getSSHWrapperFilename());
 }
 
 function getSSHWrapperFilename() {
diff --git a/node_modules/desktop-trampoline/package.json b/node_modules/desktop-trampoline/package.json
index 0cc0262..83efb47 100644
--- a/node_modules/desktop-trampoline/package.json
+++ b/node_modules/desktop-trampoline/package.json
@@ -11,7 +11,7 @@
   "types": "./index.d.ts",
   "scripts": {
     "build": "node-gyp rebuild",
-    "install": "prebuild-install || node-gyp rebuild",
+    "install": "prebuild-install",
     "test": "jest",
     "lint": "prettier -c **/*.js **/*.md",
     "lint:fix": "prettier --write **/*.js **/*.md",
@@ -30,8 +30,7 @@
   },
   "homepage": "https://github.com/desktop/desktop-trampoline#readme",
   "dependencies": {
-    "node-addon-api": "^3.1.0",
-    "prebuild-install": "^6.0.0"
+    "node-addon-api": "^3.1.0"
   },
   "devDependencies": {
     "jest": "^26.4.2",
@@ -48,5 +47,12 @@
   "config": {
     "runtime": "napi",
     "target": 3
-  }
+  },
+  "files": [
+    "index.js",
+    "index.d.ts",
+    "package.json",
+    "README.md",
+    "LICENSE"
+  ]
 }
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-arm64/desktop-trampoline b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/desktop-trampoline
new file mode 100755
index 0000000..c7e3356
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/desktop-trampoline differ
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-arm64/ssh-wrapper b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/ssh-wrapper
new file mode 100755
index 0000000..ac3dd7d
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-arm64/ssh-wrapper differ
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-x64/desktop-trampoline b/node_modules/desktop-trampoline/prebuilds/darwin-x64/desktop-trampoline
new file mode 100755
index 0000000..5d20e45
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-x64/desktop-trampoline differ
diff --git a/node_modules/desktop-trampoline/prebuilds/darwin-x64/ssh-wrapper b/node_modules/desktop-trampoline/prebuilds/darwin-x64/ssh-wrapper
new file mode 100755
index 0000000..b174748
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/darwin-x64/ssh-wrapper differ
diff --git a/node_modules/desktop-trampoline/prebuilds/linux-x64/desktop-trampoline b/node_modules/desktop-trampoline/prebuilds/linux-x64/desktop-trampoline
new file mode 100755
index 0000000..a349686
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/linux-x64/desktop-trampoline differ
diff --git a/node_modules/desktop-trampoline/prebuilds/linux-x64/ssh-wrapper b/node_modules/desktop-trampoline/prebuilds/linux-x64/ssh-wrapper
new file mode 100755
index 0000000..3fd2c72
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/linux-x64/ssh-wrapper differ
diff --git a/node_modules/desktop-trampoline/prebuilds/win32-x64/desktop-trampoline.exe b/node_modules/desktop-trampoline/prebuilds/win32-x64/desktop-trampoline.exe
new file mode 100644
index 0000000..6dc249f
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/win32-x64/desktop-trampoline.exe differ
diff --git a/node_modules/desktop-trampoline/prebuilds/win32-x64/ssh-wrapper.exe b/node_modules/desktop-trampoline/prebuilds/win32-x64/ssh-wrapper.exe
new file mode 100644
index 0000000..3d89728
Binary files /dev/null and b/node_modules/desktop-trampoline/prebuilds/win32-x64/ssh-wrapper.exe differ
