diff --git a/node_modules/desktop-trampoline/index.js b/node_modules/desktop-trampoline/index.js
index 0891bae..249ff75 100644
--- a/node_modules/desktop-trampoline/index.js
+++ b/node_modules/desktop-trampoline/index.js
@@ -1,12 +1,10 @@
+const os = require('os');
 const Path = require('path')
 
+const  prebuildTargetPath =  () => require.resolve(`@hackolade/desktop-trampoline-${os.platform()}-${os.arch()}`);
+
 function getDesktopTrampolinePath() {
-  return Path.join(
-    __dirname,
-    'build',
-    'Release',
-    getDesktopTrampolineFilename()
-  )
+  return Path.join(prebuildTargetPath(), getDesktopTrampolineFilename());
 }
 
 function getDesktopTrampolineFilename() {
@@ -16,7 +14,7 @@ function getDesktopTrampolineFilename() {
 }
 
 function getSSHWrapperPath() {
-  return Path.join(__dirname, 'build', 'Release', getSSHWrapperFilename())
+  return Path.join(prebuildTargetPath(), getSSHWrapperFilename());
 }
 
 function getSSHWrapperFilename() {
