diff --git a/node_modules/kerberos-plugins/binding.gyp b/node_modules/kerberos-plugins/binding.gyp
index 43c7d9a..f254919 100644
--- a/node_modules/kerberos-plugins/binding.gyp
+++ b/node_modules/kerberos-plugins/binding.gyp
@@ -6,6 +6,18 @@
       'sources': [
         'src/kerberos.cc'
       ],
+      'xcode_settings': {
+        'MACOSX_DEPLOYMENT_TARGET': '10.12',
+        'OTHER_CFLAGS': [
+          "-std=c++17",
+          "-stdlib=libc++"
+        ],
+      },
+      'msvs_settings': {
+        'VCCLCompilerTool': {
+          'AdditionalOptions': [ '-std:c++17', ],
+        },
+      },
       'conditions': [
         ['OS=="mac" or OS=="linux"', {
           'sources': [
@@ -32,16 +44,6 @@
               'Shlwapi.lib'
             ]
           }
-        }],
-        ['OS=="mac"', {
-          'cflags!': [ '-fno-exceptions' ],
-          'cflags_cc!': [ '-fno-exceptions' ],
-          'xcode_settings': {
-            'GCC_ENABLE_CPP_RTTI': 'YES',
-            'OTHER_CPLUSPLUSFLAGS' : [ '-std=c++11', '-stdlib=libc++' ],
-            'OTHER_LDFLAGS': [ '-stdlib=libc++' ],
-            'MACOSX_DEPLOYMENT_TARGET': "10.7"
-          }
         }]
       ]
     }
diff --git a/node_modules/kerberos-plugins/lib/kerberos.js b/node_modules/kerberos-plugins/lib/kerberos.js
index 09f8425..15806f6 100644
--- a/node_modules/kerberos-plugins/lib/kerberos.js
+++ b/node_modules/kerberos-plugins/lib/kerberos.js
@@ -1,6 +1,7 @@
 'use strict';
 
-const kerberos = require('bindings')('kerberos');
+const os = require('os');
+const kerberos = require(`@hackolade/kerberos-plugins-${os.platform()}-${os.arch()}/prebuild.node`);
 const KerberosClient = kerberos.KerberosClient;
 const KerberosServer = kerberos.KerberosServer;
 const defineOperation = require('./util').defineOperation;
