diff --git a/node_modules/mongodb-client-encryption/.npmignore b/node_modules/mongodb-client-encryption/.npmignore
new file mode 100644
index 0000000..6dc8851
--- /dev/null
+++ b/node_modules/mongodb-client-encryption/.npmignore
@@ -0,0 +1,3 @@
+build
+prebuilds
+src
diff --git a/node_modules/mongodb-client-encryption/README.md b/node_modules/mongodb-client-encryption/README.md
index ab27f0e..007f6a2 100644
--- a/node_modules/mongodb-client-encryption/README.md
+++ b/node_modules/mongodb-client-encryption/README.md
@@ -1,3 +1,10 @@
+# Repackaged version of mongodb-client-encryption
+
+This is a convenience package that will install all prebuilds for all supported platforms, to the contrary to the upstream package that constraint the installation only to the runtime platform (os/cpu).
+This typically enable cross building an Electron application.
+
+The package is cleaned up to only embbed the necessary.
+
 MongoDB Client Encryption
 =========================
 
diff --git a/node_modules/mongodb-client-encryption/lib/autoEncrypter.js b/node_modules/mongodb-client-encryption/lib/autoEncrypter.js
index f4e3dc0..6a70d90 100644
--- a/node_modules/mongodb-client-encryption/lib/autoEncrypter.js
+++ b/node_modules/mongodb-client-encryption/lib/autoEncrypter.js
@@ -1,7 +1,8 @@
 'use strict';
 
 module.exports = function (modules) {
-  const mc = require('bindings')('mongocrypt');
+  const os = require('os');
+  const mc = require(/* webpackIgnore: true */ `@hackolade/mongodb-client-encryption-${os.platform()}-${os.arch()}`);
   const common = require('./common');
   const databaseNamespace = common.databaseNamespace;
   const StateMachine = modules.stateMachine.StateMachine;
diff --git a/node_modules/mongodb-client-encryption/lib/clientEncryption.js b/node_modules/mongodb-client-encryption/lib/clientEncryption.js
index 10bc541..c5fab91 100644
--- a/node_modules/mongodb-client-encryption/lib/clientEncryption.js
+++ b/node_modules/mongodb-client-encryption/lib/clientEncryption.js
@@ -1,7 +1,8 @@
 'use strict';
 
 module.exports = function (modules) {
-  const mc = require('bindings')('mongocrypt');
+  const os = require('os');
+  const mc = require(/* webpackIgnore: true */ `@hackolade/mongodb-client-encryption-${os.platform()}-${os.arch()}`);
   const common = require('./common');
   const databaseNamespace = common.databaseNamespace;
   const collectionNamespace = common.collectionNamespace;
diff --git a/node_modules/mongodb-client-encryption/package.json b/node_modules/mongodb-client-encryption/package.json
index e5f5a34..9b48b6d 100644
--- a/node_modules/mongodb-client-encryption/package.json
+++ b/node_modules/mongodb-client-encryption/package.json
@@ -9,14 +9,13 @@
     "CHANGELOG.md",
     "lib",
     "src",
-    "index.d.ts",
-    "binding.gyp"
+    "index.d.ts"
   ],
   "directories": {
     "lib": "lib"
   },
   "scripts": {
-    "install": "prebuild-install --runtime napi --tag-prefix node-v || node-gyp rebuild",
+    "install": "prebuild-install --runtime napi --tag-prefix node-v",
     "check:lint": "eslint lib test && tsd",
     "docs": "jsdoc2md --template etc/README.hbs --plugin dmd-clear --files 'lib/**/*.js' > README.md",
     "test": "mocha test",
@@ -34,9 +33,7 @@
   "license": "Apache-2.0",
   "gypfile": true,
   "dependencies": {
-    "bindings": "^1.5.0",
     "node-addon-api": "^4.3.0",
-    "prebuild-install": "^7.1.1",
     "socks": "^2.7.1"
   },
   "devDependencies": {
@@ -55,6 +52,7 @@
     "mongodb": "^5.3.0",
     "node-gyp": "^9.3.1",
     "prebuild": "^11.0.4",
+    "prebuild-install": "^7.1.1",
     "prettier": "^2.8.8",
     "sinon": "^15.0.4",
     "sinon-chai": "^3.7.0",
diff --git a/node_modules/mongodb-client-encryption/prebuilds/darwin-arm64/mongocrypt.node b/node_modules/mongodb-client-encryption/prebuilds/darwin-arm64/mongocrypt.node
new file mode 100755
index 0000000..f09b75f
Binary files /dev/null and b/node_modules/mongodb-client-encryption/prebuilds/darwin-arm64/mongocrypt.node differ
diff --git a/node_modules/mongodb-client-encryption/prebuilds/darwin-x64/mongocrypt.node b/node_modules/mongodb-client-encryption/prebuilds/darwin-x64/mongocrypt.node
new file mode 100755
index 0000000..04ea074
Binary files /dev/null and b/node_modules/mongodb-client-encryption/prebuilds/darwin-x64/mongocrypt.node differ
diff --git a/node_modules/mongodb-client-encryption/prebuilds/linux-x64/mongocrypt.node b/node_modules/mongodb-client-encryption/prebuilds/linux-x64/mongocrypt.node
new file mode 100755
index 0000000..9a2d3be
Binary files /dev/null and b/node_modules/mongodb-client-encryption/prebuilds/linux-x64/mongocrypt.node differ
diff --git a/node_modules/mongodb-client-encryption/prebuilds/win32-x64/mongocrypt.node b/node_modules/mongodb-client-encryption/prebuilds/win32-x64/mongocrypt.node
new file mode 100644
index 0000000..a407635
Binary files /dev/null and b/node_modules/mongodb-client-encryption/prebuilds/win32-x64/mongocrypt.node differ
